libs/packages/components/src/lib/search-result-list/search-result-list.module.ts

```diff
@@ -4,14 +4,11 @@ import { FormsModule } from '@angular/forms';
 import { SdsSearchResultListComponent } from './search-result-list.component';
 
 import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+import { SdsIconModule } from '../icon/icon.module';
 
 @NgModule({
-  imports: [
-    CommonModule, FormsModule, FontAwesomeModule
-  ],
+  imports: [CommonModule, FormsModule, FontAwesomeModule, SdsIconModule],
```

Comment:
I didn't see fa-icons used in the html now. Would it be fine to remove the FontAwesome dependency?

Reply:
Removed module